types/stack/branch/merge-queue.d.ts

```diff
@@ -0,0 +1,6 @@
+import { Response } from "../../contentstackCollection"
+
+export interface MergeQueue {
+  fetch():Promise<Response>
```

Comment:
@nadeem-cs add two interface `MergeQueue` with `fetch` function and `MergeQueues` with `find` function allowing `params` as optional arg.
`fetch` and `find` should return `Promise<AnyProperty>`.